app/controllers/candidate_interface/work_history/length_controller.rb

```diff
@@ -0,0 +1,27 @@
+module CandidateInterface
+  class WorkHistory::LengthController < CandidateInterfaceController
+    def length
```

Comment:
not sure about this name, from the code bellow`@work_details_form = WorkHistoryForm.new` it looks like it's about `edit_<something>`

Reply:
Fixed, it's `show` now 👍